hops/util.py

```diff
@@ -111,6 +114,50 @@ def _get_host_port_pair():
     return host_port_pair
 
 
+def connect(host=None, port=443, scheme="https", hostname_verification=False,
+            api_key=None,
```

Comment:
This should be `api_key_file` otherwise user thinks that you can pass the API key directly as parameter.

Reply:
@tkakantousis Does it have to be like this? For example in my case I read the API key from a configuration file with other properties too. A common technique is to export it to environment variable. Was it always like that or was it introduced in this PR?

Reply:
@SirOibaf done
@kouzant yes it has been like that since it's first implementation https://github.com/logicalclocks/hops-util-py/blob/master/hops/util.py#L445 I would prefer to fix it in another PR

Reply:
ok ;)